dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -250,3 +262,26 @@ CWPKI2056W=CWPKI2056W: The validity period of the certificate with {0} serial nu
 CWPKI2056W.explanation=The validity period is shorter than the minimum renew time. The certificate expires before a new certificate is requested.
 CWPKI2056W.useraction=To avoid certificate expiration, request a certificate with a longer validity period. If the certificate authority supports a custom validity period, set the validFor property in the server configuration.
 
+CWPKI2057E=CWPKI2057E: Certificate revocation status checking failed to create a CertPathValidator to validate the certificate. The error is ''{0}}''.
+CWPKI2057E.explanation=The certificate revocation checker needs to build a CertPathValidator to check OCSP and / or CRLs revocation status.
+CWPKI2057E.useraction=Review the status message and error for details on the failure.
+
+CWPKI2058W=CWPKI2058W: Certificate revocation status checking ignored soft failures. Revocation checking may be incomplete. The failures are: ''{0}''.
+CWPKI2058W.explanation=Soft failures include network errors. The ACME service ignores soft errors as they may be temporary glitches. 
+CWPKI2058W.useraction=If this message is encountered consistently, the cause of the failure should be resolved as it may prevent proper revocation status checking. Otherwise; this warning can be safely ignored.  
+
+CWPKI2059I=CWPKI2059I: Certificate revocation status checking found that the certificate with the {0} serial number has been revoked.
+CWPKI2059I.explanation=The ACME service found that the certificate was marked revoked by either a CRL or OSCP responder. 
+CWPKI2059I.useraction=No action is required.
+
+CWPKI2060E=CWPKI2060E: Failed to retrieve the OCSP URL from the certificate with the {0} serial number. The error is: ''{1}''. 
+CWPKI2060E.explanation=The OCSP URL was not retrieved.
+CWPKI2060E.useraction=Ensure that the certificate is a valid X.509 certificate. If it is not valid, request a new certificate.
+
+CWPKI2061E=CWPKI2061E: Failed to retrieve the CRL distribution points from the certificate with the {0} serial number. The error is ''{1}''.
```

Comment:
Change,
"Failed to retrieve the CRL distribution points from the certificate with the {0} serial number. The error is ''{1}''."
to,
"The CRL distribution points from the certificate with the {0} serial number wasn't retrieved. The error is ''{1}''."

Reply:
Changed, but with `wasn't` --> `weren't`.